packages/solid/components/Row/Row.types.ts

```diff
@@ -53,6 +49,11 @@ export interface RowProps extends UIComponentProps, RowStyleProperties {
     selectedIndex: number,
     lastSelectedIndex: number
   ) => void;
+
+  /**
+   * Wrap the row so active goes back to the beginning of the row
+   */
+  wrap: boolean;
```

Comment:
I'm realizing this is used regardless of direction in handleNavigation, can we add this to the Column types as well?